ml-algorithms/build.gradle

```diff
@@ -65,7 +65,10 @@ dependencies {
     implementation platform('software.amazon.awssdk:bom:2.25.40')
     api 'software.amazon.awssdk:auth:2.25.40'
     implementation 'software.amazon.awssdk:apache-client'
-    implementation 'com.amazonaws:aws-encryption-sdk-java:2.4.1'
+    implementation ('com.amazonaws:aws-encryption-sdk-java:2.4.1') {
+        exclude group: 'org.bouncycastle', module: 'bcprov-ext-jdk18on'
```

Comment:
These are separate modules, but it looks like they may have been merged based on the message from https://mvnrepository.com/artifact/org.bouncycastle/bcprov-ext-jdk18on.


> This artifact was moved to:

> [org.bouncycastle](https://mvnrepository.com/artifact/org.bouncycastle) » [bcprov-jdk18on](https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk18on)


One is `bcprov-ext-jdk18on` while the other is `bcprov-jdk18on`.

Reply:
Do you see any concern?

Reply:
This encryption SDK used to encrypt/decrypt connector credential. Have you tested any regression issue by upgrading to `bcprov-jdk18on` ? Like can the old connector's credential be decrypted correctly?

Reply:
Yup, I just tested as we discussed and the prediction works as expected with the older connected created by the version `bcprov-ext-jdk18on`